apps/backend/src/libs/enums/relation-name.enum.ts

```diff
@@ -1,8 +1,10 @@
 const RelationName = {
 	ONBOARDING_ANSWERS: "answers",
```

Comment:
Why do we have two distinct relations with the same value (ONBOARDING_ANSWERS, QUIZ_ANSWERS)?

Reply:
On quiz question and onboarding questions model, there's a relationship mapping named answer for both model. To avoid confusion on seeing relation name labelled with just ANSWER, I think it would make more sense to have it labelled separately.